TwitchConnect.js

```diff
@@ -62,74 +70,37 @@ twitchClient.on("message", (channel, tags, message, self) => {
   // Ignore echoed messages.
   if (self) return;
 
-  //Sell Stonks
-  if (
-    message.toLowerCase().includes("corpa sell") ||
-    message.toLowerCase().includes("sell corpa")
-  ) {
-    var streamerName = channel.toString().toLowerCase().substring(1);
-    netWorthMap.set(
-      channel.toString().substring(1),
-      netWorthMap.get(channel.toString().substring(1)) - 0.01
-    );
-    insertCorpa(
-      new MongoClient(
-        process.env.MONGO_URL.concat(channel.toString().substring(1))
-      ),
-      streamerName,
-      {
-        channelId: channel,
-        stockName: "$".concat(channel.toString().substring(1)),
-        stockPrice: netWorthMap.get(channel.toString().substring(1)),
-        //timestamp: new Date().toISOString(),
-        timestamp: Math.round(Date.now() / 1000),
-      }
-    );
-  }
-
-  //Buy Stonks
-  if (
-    message.toLowerCase().includes("corpa buy") ||
-    message.toLowerCase().includes("buy corpa")
-  ) {
+  // Add/Sub corpa value
+  if (keywords.includes(message.toLowerCase())) {
```

Comment:
imo, split this condition into 2 sell/buy. You make this same check down in line 77 and 78, so it's kind of redundant.

Reply:
In order for me to split this id have to split my `keywords` array to `sellKeywords `and `buyKeywords`. This would eliminate the need to recheck on lines 77 and 78